pmm-app/tests/verifyPMMDemoDashboards_test.js

```diff
@@ -0,0 +1,193 @@
+Feature('Tests for PMM Demo Sanity Tests Cycle Dashboard Checks');
+
+Scenario(
+  // eslint-disable-next-line max-len
+  'PMM-T319 Open the MySQL Instance Overview Dashboard and verify Metrics are present and graphs are displayed [critical] @pmm-demo @not-ui-pipeline @not-pr-pipeline',
+  async (I, adminPage, dashboardPage, pmmDemoPage) => {
+    const serviceName = ['ps8-mysql', 'pxc57-3-mysql', 'ps8-slave-mysql'];
+    I.amOnPage(pmmDemoPage.url + dashboardPage.mySQLInstanceOverview.url);
+    dashboardPage.waitForDashboardOpened();
+    adminPage.peformPageDown(5);
+    await dashboardPage.expandEachDashboardRow();
+    for (i = 0; i < serviceName.length; i++) {
+      await dashboardPage.applyFilter('Service Name', serviceName[i]);
+      I.click(adminPage.fields.metricTitle);
+      adminPage.peformPageDown(5);
+      dashboardPage.verifyMetricsExistence(dashboardPage.mySQLInstanceOverview.metrics);
+      await dashboardPage.verifyThereAreNoGraphsWithNA();
+      await dashboardPage.verifyThereAreNoGraphsWithoutData(1);
+    }
+  }
+);
+
+Scenario(
+  // eslint-disable-next-line max-len
+  'PMM-T319 Open the MySQL Summary Dashboard and verify Metrics are present and graphs are displayed [critical] @pmm-demo @not-ui-pipeline @not-pr-pipeline',
+  async (I, adminPage, dashboardPage, pmmDemoPage) => {
+    const serviceName = ['ps8-mysql', 'pxc57-3-mysql', 'ps8-slave-mysql'];
+    I.amOnPage(pmmDemoPage.url + dashboardPage.mysqlInstanceSummaryDashboard.url);
+    dashboardPage.waitForDashboardOpened();
+    adminPage.peformPageDown(5);
+    await dashboardPage.expandEachDashboardRow();
+    for (i = 0; i < serviceName.length; i++) {
+      await dashboardPage.applyFilter('Service Name', serviceName[i]);
+      I.click(adminPage.fields.metricTitle);
+      adminPage.peformPageDown(5);
+      dashboardPage.verifyMetricsExistence(dashboardPage.mysqlInstanceSummaryDashboard.metrics);
+      await dashboardPage.verifyThereAreNoGraphsWithNA();
+      await dashboardPage.verifyThereAreNoGraphsWithoutData(2);
+    }
+  }
+);
+
+Scenario(
+  'PMM-T69 Open the PostgreSQL Instance Summary Dashboard and verify Metrics are present and graphs are displayed [critical] @pmm-demo @not-ui-pipeline @not-pr-pipeline',
```

Comment:
Don't we have to put the `// eslint-disable-next-line max-len` here? Or the linter is not being run on tests?

Reply:
it is run on the tests, I will add it thanks.